docs/ml-guide.md

```diff
@@ -455,6 +465,14 @@ val pipeline = new Pipeline()
 // Fit the pipeline to training documents.
 val model = pipeline.fit(training)
 
+// now we can optionally save it to disk
+model.save("/tmp/spark-logistic-regression-model")
+
+// and load it back in during production
+val loadedModel = Pipeline.load("/tmp/spark-logistic-regression-model")
+// or equivalently
```

Comment:
I like the idea in this & the next line, but I'd remove this since it will cause problems if users blindly copy and paste the code into a spark shell.